app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -114,7 +115,9 @@ fun MapScreen(
             .pitch(0.0)
             .zoom(maxZoom)
             .padding(EdgeInsets(100.0, 100.0, 100.0, 100.0))
-            .build())
+            .build()) {
+          mapViewportState.cameraState?.let { mapViewModel.updateUserPosition(it.center) }
```

Comment:
don't forget to update the documentation a bit since you are updating the userPosition value in MapViewModel.